### ISSUE TITLE
Content Test - contains check

### ISSUE THREAD

**Issue by thim81** [apideck-libraries/portman#158]
@benjaminDC suggested to extend the content test validation with a "contains" check. 

The current "value" check for content test, does a strict check verifying if the value matches. The "contains" check would do a fluid check, verifying if the string is present in the value to test. 

Example:
```json
{
"contentTests": [
      {
        "openApiOperationId": "post-oauth-request-token",
        "responseBodyTests": [
          {
            "key": "token_type",
            "value": "Bearer"
          },
          {
            "key": "scope",
            "contains": "broadcast:push"
          }
        ]
      }
    ]
}
```

Where the "scope" would return a whole range of scopes, like "send:sms send:email broadcast:push"

**Comment by thim81** [comment]
This will be part of the 1.10 release.

**Comment by thim81** [comment]
Closing this request, since the feature is included in the 1.10.0 release
